terraform/main.tf

```diff
@@ -213,6 +221,11 @@ resource "google_cloudbuild_trigger" "update-schema" {
     "_NAME" : google_sql_database.db.name
     "_SSLMODE" : "disable"
   }
+  depends_on = [google_project_iam_member.cloudbuild-secrets, google_project_iam_member.cloudbuild-sql]
+}
+
+# "build" does first time setup - it is different from "deploy" which we set up to trigger for later.
```

Comment:
Oh, copy-paste error here, this comment applies to the below null resource.

Reply:
Thanks! Fixed.